### PR TITLE
adding user info to other requests

### PR DESCRIPTION
**Pull request by marinaRomanchuk**

### CODE REVIEW

File: comments/serializers.py [hunk 1]
```diff
@@ -3,12 +3,15 @@
 from rest_framework import serializers
 
 from comments.models import Comment
+from users.serializers import UserPublicSerializer
 
 
 class CommentSerializer(serializers.ModelSerializer):
+    user_info = UserPublicSerializer(source="user", required=False)
```
Comment (revimi) [inline comments/serializers.py:10]:
```suggestion
    user = UserPublicSerializer(source="user", required=False)
```
or
```suggestion
    author = UserPublicSerializer(source="user", required=False)
```

**Review by revimi — APPROVED** [review]
You can also add post stats to the endpoints where you get posts, so that you do less requests to get the data for the feed

File: posts/serializers.py [hunk 2]
```diff
@@ -4,14 +4,16 @@
...
 
 
 class PostSerializer(serializers.ModelSerializer):
     photo = Base64ImageField()
+    user_info = UserPublicSerializer(source="user", required=False)
 
     class Meta:
         model = Post
-        fields: list = ["id", "photo", "description", "timestamp", "user_id"]
+        fields: list = ["id", "photo", "description", "timestamp", "user_info"]
```
Comment (revimi) [inline posts/serializers.py]:
```suggestion
        fields: list = ["id", "photo", "description", "timestamp", "user"]
```
or
```suggestion
        fields: list = ["id", "photo", "description", "timestamp", "author"]
```

File: comments/serializers.py (hunk 1 above)
Comment (marinaRomanchuk) [reply comments/serializers.py:10]: Done

File: posts/serializers.py (hunk 2 above)
Comment (marinaRomanchuk) [reply posts/serializers.py]: Done

**Review by revimi — APPROVED** [review]

**Pull request merged**